canal/dump.go

```diff
@@ -185,8 +186,8 @@ func (c *Canal) dump() error {
 		c.master.UpdateGTIDSet(h.gset)
 		startPos = h.gset
 	}
-	c.cfg.Logger.Infof("dump MySQL and parse OK, use %0.2f seconds, start binlog replication at %s",
-		time.Since(start).Seconds(), startPos)
+	c.cfg.Logger.Info(fmt.Sprintf("dump MySQL and parse OK, use %0.2f seconds, start binlog replication at %s",
+		time.Since(start).Seconds(), startPos))
```

Comment:
Shouldn't we put the time and position in separate fields?

Something like this, maybe?
```suggestion
	c.cfg.Logger.Info("dump MySQL and parse OK", "use", slog.Time(time.Since(start)), "position", startPos)
```

Or keep the fmt.Sprintf, but add the fields as well?

Reply:
`time.Since` returns a `time.Duration` while `slog.Time` expects `time.Time`. Will update to pass fields